src/__swaps__/screens/Swap/hooks/useSwapSettings.ts

```diff
@@ -14,6 +14,16 @@ export const useSwapSettings = ({ inputAsset }: { inputAsset: SharedValue<Extend
   const setSlippage = swapsStore(state => state.setSlippage);
   const setFlashbots = swapsStore(state => state.setFlashbots);
 
+  const checkIfFlashbotsIsEnabledOnSelectedInputAssetChain = (asset: ExtendedAnimatedAssetWithColors | null) => {
+    'worklet';
+
+    if (asset?.chainId === ChainId.mainnet) {
```

Comment:
we should avoid network-specific logic like this - the source of truth for if flashbots is enabled is in the `networks` config (under `features` -> `flashbots`)